src/styles/moyo-logo.scss

```diff
@@ -0,0 +1,91 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.logoimage {
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  box-sizing: border-box;
+  padding: 0 50px;
+  box-shadow: 0px 2px 4px 0px #0000000D;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  flex-wrap: nowrap;
+  text-transform: uppercase;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  height: 60px;
+  line-height: 60px;
+  font-weight: 500;
+}
+
+.menu-item {
+  font-size: 12px;
+  margin: 0 10px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  color: #060B35;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+  height: 100%;
+  transition: color $anim-duration ease;
```

Comment:
The variable `$anim-duration` is used but not defined. Please define this variable to avoid compilation errors.